tests/packages/media-sound/bitwig-studio-1.3.16.conf

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cat >/etc/portage/package.use/bitwig-studio <<EOF
```

Comment:
Actually I don't like the echo "..." > /adsfasd syntax here.
If I want to add something to the file, I have to rewrite it.